### DISCUSSION TITLE
What should we do about events that have been organized?

### DISCUSSION THREAD

**Discussion by kaf-lamed-beyt** [kaf-lamed-beyt/ng-tech-events#4] [Ideas]
I've been thinking about what to do concerning events that have been held, and are still in the README.

My first option was to remove the particular event(s) from the repo, since it's been organised already. But then, a new thought came to mind. 

What if we categorize these events by year, such that we update the header to include, not just the location (states), but the year also. 

What do you think?

cc: @Jemeni11

**Comment by Jemeni11** [comment]
Hey
> What if we categorize these events by year, such that we update the header to include, not just the location (states), but the year also.
>
> What do you think?

You mean something like:

<h1 align="center">#ng-tech-events 💡</h1>

<p align="center">
   <a href="#">2023</a> &#183; <a href="#">2022</a> &#183; <a href="#">2021</a><br/>
   <a href="#abuja">Abuja</a> &#183; <a href="#ilorin">Ilorin</a> &#183; <a href="#lagos">Lagos</a> &#183; <a href="#portharcourt">Port Harcourt</a>
</p>
<p align="center">A pool of upcoming and perhaps old tech events happening 9ja's tech ecosystem.</p>

# 2023
## January
### Ilorin
- **Polygon Ilorin &mdash; Introduction to Web3**
  Introduction to web3 and the official launch of the Polygon Guild Ilorin
  Date: Saturday 28th, January 2023
  Venue: Peace Hotel
  [Register here](https://docs.google.com/forms/d/1k7xP-i4WOSFwmkH9WRC2gUtBTbnIj65n5H32-mqMNTM/viewform?edit_requested=true)
## February
###  Ilorin
- **Practiced Curiosity: Building Collaboration Between Development and Design**
  Date: Saturday 4th, February 2023
  Venue: De Peace Hotel & Suites, Tanke Rd, 240102, Ilorin.
  [Register here](bit.ly/ACECollaboration)
## March
## April
## May
## June
### Lagos
- **OSCAFEST &mdash; Opensource Festival &mdash; 2023**
  An annual open source gathering under Opensource Community Africa
  Date: Thursday 22nd June 2023 - Saturday 24th June 2023 
  Venue: Lagos, Nigeria

**Reply by kaf-lamed-beyt** [reply]
Yeah yeah... something like this, but without the months

**Reply by Jemeni11** [reply]
This is cool
The only problem I foresee is the location links (Lagos, Ibadan, ... )
If we change it to year first, how will those links work since they are basically IDs now?

**Comment by kaf-lamed-beyt** [comment]
This is a great observation!!

**Comment by kaf-lamed-beyt** [comment]
Okay... what if we just have a text above the fold, that links to past events below the fold — so, what we'll do here is, push all the events that have been organized, down in the README.

How does this sound?

**Reply by Jemeni11** [reply]
Sounds great :rocket:

**Comment by kaf-lamed-beyt** [comment]
Awesome! 

I mentioned you on an issue (#5). Would you love to take it up?

**Reply by Jemeni11** [reply]
I'd love to
I'm done with it :+1:

**Reply by kaf-lamed-beyt** [reply]
🚀🚀🚀

**Comment by kaf-lamed-beyt** [comment]
@Jemeni11 @Tonie-NG, I think there are some physical 9ja-based events [here](https://www.diversifytech.com/events)

**Reply by tonievictor** [reply]
Alright, I'll check them out and do the necessary 🤞

**Reply by tonievictor** [reply]
I've just gone through it, there's no 9ja based physical event

**Reply by kaf-lamed-beyt** [reply]
Oh wow! I thought I saw some GDG Women Techmakers event sometime ago

**Reply by tonievictor** [reply]
Sure about that? I'll check again

**Reply by kaf-lamed-beyt** [reply]
I don't even know. But, I'm sure I saw those events the last tike I checked. 

Maybe they've removed it? Or probably because the event have all been held?
